src/cnaas_nms/app_settings.py

```diff
@@ -38,14 +37,14 @@ class ApiSettings(BaseSettings):
     HTTPD_URL: str = "https://cnaas_httpd:1443/api/v1.0/firmware"
     VERIFY_TLS: bool = False
     VERIFY_TLS_DEVICE: bool = False
-    JWT_CERT: Path = "/opt/cnaas/jwtcert/public.pem"
-    CAFILE: Optional[Path] = "/opt/cnaas/cacert/rootCA.crt"
-    CAKEYFILE: Path = "/opt/cnaas/cacert/rootCA.key"
-    CERTPATH: Path = "/tmp/devicecerts/"
+    JWT_CERT: Path = Path("./opt/cnaas/jwtcert/public.pem")
```

Comment:
The path here has an extra . at the beginning causing integration test errors

Reply:
Thanks!